samples/tcp_server.cr

```diff
@@ -20,4 +20,7 @@ end
 
 server = TCPServer.new "127.0.0.1", 9000
 puts "listen on 127.0.0.1:9000"
-loop { spawn process(server.accept) }
+loop do
+  client = server.accept
+  spawn { process(client) }
```

Comment:
Exactly, that's what I said in my previous comment, but I think it's acceptable that there might be common pitfalls until someone learns how things work.